CONTRIBUTING.md

```diff
@@ -0,0 +1,146 @@
+
+## Coding Style
+
+The code in this package is also automatically formatted by `black` for consistency.
+
+## The Development Life Cycle
+
+### Fork and clone this repository
+
+1. See the [Github docs](https://help.github.com/articles/fork-a-repo/) for how to make a copy (a fork) of a repository to your own Github account.
+1. Then, [clone the repository](https://help.github.com/articles/cloning-a-repository/) to your local machine so you can begin making changes.
+1. Add this repository as an [upstream remote](https://help.github.com/en/articles/configuring-a-remote-for-a-fork) on your local git repository so that you are able to fetch the latest commits.
+1. On your local machine make sure you have the latest version of the `develop` branch:
+
+    ```shell
+    git checkout develop
+    git pull upstream develop
+    ```
+
+### Install development dependencies
+
+Please follow the [README.md](README.md) to install the package for development purposes.
+
+### Developing at Sage Bio
+
+The agora-data-tools project follows the standard [trunk based development](https://trunkbaseddevelopment.com/) development strategy.
+> To ensure the most fluid development, do not push to your `dev`
+
+1. Access
+    1. (Assuming you have followed all 4 steps above in the "fork and clone this repository" section). Navigate to your cloned repository on your computer/server. 
+    1. If you are Sage Bio employee working directly on this project, please ask for write permissions to contribute to the repository.
+1. Make sure you are always creating feature branches from the `dev` branch.
+
+    ```shell
+    # On a fork
+    cd {your-github-username}/Genie
+    git checkout develop
+    git pull upstream develop
+    # Write access
+    git checkout develop
+    git pull
+    ```
+
+1. Create a feature branch which off the `dev` branch. If there is JIRA issue that you are addressing, name the branch after the issue with some more detail (like `{user}/{GH|JIRA}-123/add-some-new-feature`).
+
+    ```shell
+    git checkout develop
+    git checkout -b tyu/JIRA-123/some-new-feature
+    ```
+
+1. At this point, you have only created the branch locally, you need to push this to your fork on GitHub.
+
+    ```shell
+    git push --set-upstream origin tyu/JIRA-123/some-new-feature
+    ```
+
+    You should now be able to see the branch on GitHub. Make commits as you deem necessary. It helps to provide useful commit messages - a commit message saying 'Update' is a lot less helpful than saying 'Remove X parameter because it was unused'.
+
+    ```shell
+    git commit changed_file.txt -m "Remove X parameter because it was unused"
+    git push
+    ```
+
+1. (Make sure you have follow instructions in "Install development dependencies") Once you have made your additions or changes, make sure you write tests and run the test suite.  More information on testing below.
+
+    ```shell
+    pytest ./test
```

Comment:
Why is this `pytest ./test` but below it's `pytest -vs tests/`? I think the first one (here) wouldn't work?